geometry/segment2d.cpp

```diff
@@ -40,8 +42,61 @@ bool IsPointOnSegment(PointD const & pt, PointD const & p1, PointD const & p2)
   return IsPointOnSegmentEps(pt, p1, p2, kEps);
 }
 
+bool AreSegmentsIntersected(PointD const & a, PointD const & b, PointD const & c, PointD const & d)
```

Comment:
s/AreSegmentsIntersected/SegmentsIntersect/

(otherwise a question "by what?" is begging to be asked)

Reply:
Done